client/package.json

```diff
@@ -209,7 +209,8 @@
           "regexp": "^.*ParseError (at|in) (.+?):(\\d+)?:? (.+)$",
```

Comment:
Have you tested or had a look at the other kinds of errors? Do they have the same problem?

Reply:
yes, I put the comments on this in the ticket. I will put them here.

> Different errors can have slightly different logs in terms of the log format.
python function error:
![image](https://github.com/yoctoproject/vscode-bitbake/assets/47988425/342c68b2-6daf-4cc4-a006-a791e7ea352f)
Normal line error:
![image](https://github.com/yoctoproject/vscode-bitbake/assets/47988425/dc978b7b-e29b-4fa5-874a-994a9326bc4a)